CADD.sh

```diff
@@ -14,6 +14,7 @@ where:
     -q  print basic information about snakemake run
     -p  print full information about the snakemake run
     -d  do not remove temporary directory for debug puroposes
+    -t  specify location for temporary directory [default: /tmp/]
```

Comment:
I thought about that too. But in tehory it can be controlled via the variable $TMPDIR

Using you code will change the default behaviour creating always a tmp folder under /Ttmp/ and not in $TMPDIR if set (or not specified via `-d`)

Reply:
ok, that makes sense.  you might default to $TMPDIR as well, as I actually wasn't aware of $TMPDIR :-)